.github/go.mod

```diff
@@ -0,0 +1,11 @@
+module sigstore/sigstore-python
+
+go 1.18
```

Comment:
this is very old by golang standards, probably unsupported for years by now...

Likely dependabot doesn't actually use golang at all so maybe that does not matter?

Reply:
I've bumped the version here just in case and used the one in the latest ubuntu-runner.